### DISCUSSION TITLE
Big Sur Update

### DISCUSSION THREAD

**Discussion by tahaklkt-osxinfo** [chris1111/Legacy-Video-patch#35] [Ideas]
Hello friend, Big Sur is really nice and stable. As you know, we have a GT330M graphics card, but we cannot use it in Big Sur. I am waiting for your update.

**Comment by chris1111** [comment]
Sorry but I am not working on this to make it work on Big Sur

**Reply by tahaklkt-osxinfo** [reply]
Are you working on Big Sur sir ? Please im really want use big sur

**Reply by chris1111** [reply]
> Are you working on Big Sur sir ? Please im really want use big sur

No

**Reply by tahaklkt-osxinfo** [reply]
Sir, i changed my efi. I get black screen issue. How can i fix this issue ? Thanks.
